tasks/db_deploy/API.md

```diff
@@ -1,182 +1,12 @@
 # Table of Contents
```

Comment:
looks like it did not capture all the contents under install and migrations folder when I ran bin/build_doc.sh

Reply:
ok that worked! thanks